src/main/osd/osd_elements.c

```diff
@@ -1051,6 +1051,12 @@ static void osdElementMotorDiagnostics(osdElementParms_t *element)
     for (; i < getMotorCount(); i++) {
         if (motorsRunning) {
             element->buff[i] =  0x88 - scaleRange(motor[i], getMotorOutputLow(), getMotorOutputHigh(), 0, 8);
+#if defined(USE_ESC_SENSOR) || defined(USE_DSHOT_TELEMETRY)
+            if ( getEscRpm(i) < 700 ) {
+                // motor is not spinning properly. Mark as Stopped
+                element->buff[i] =  0x53; // S
```

Comment:
You can just use `S` here - this will also remove the need for a comment.

Reply:
Heh, didn't know. That is indeed a lot better. Thanks!